### PR TITLE
Add a makefile

### PR DESCRIPTION
**Pull request by NiallBunting**
## Ticket title

One line description
[link to ticket number]()

## Checklist

- [ ] Code has been tested manually
- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by tom-bjss — APPROVED** [review]

**Review by danielchildsbjss — APPROVED** [review]

**Comment by NiallBunting** [comment]
I don't have permission to merge this change.

File: makefile [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+.PHONY: all help init install build run lint test e2e plan deploy
+all: lint test run #help Full check
+
+help:
+	@echo "-- HELP --"
+	@grep '#[h]elp' makefile
+
+init: #help Run through dependencies and check
+	git secrets
+	echo "Is this node 16?"
```
Comment (danielchildsbjss) [inline makefile:12]: This should be node 14 for the microservices (see package.json) and https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

File: makefile [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	npm i
+
+build: #help Bulid the project files
+	npm run build
+
+run: #help Run Locally
+	npm run start
+
+storybook: #help Run Storybook
+	npm run storybook
```
Comment (Daniel-Searle) [inline makefile:26]: This should be removed due to Storybook being a component viewer.

File: makefile [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+.PHONY: all help init install build run lint test e2e plan deploy
+all: lint test run #help Full check
+
+help:
+	@echo "-- HELP --"
+	@grep '#[h]elp' makefile
+
+init: #help Run through dependencies and check
+	git secrets
```
Comment (Daniel-Searle) [inline makefile:11]: This command is being used incorrectly and is erroring.

File: makefile [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+	echo "Is this node 16?"
+	node -v
+	echo "If so, OK!"
+
+install: #help Install packages
+	npm i
+
+build: #help Bulid the project files
```
Comment (Daniel-Searle) [inline makefile:19]: Spelling error in build.

File: makefile [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+	npm run storybook
+
+lint: #help Run linting
+	npm run lint
+
+test: #help Run the unit tests
+	npm run test
+	npm run sonar-scanner
+
+e2e: #help Run the e2e tests
```
Comment (Daniel-Searle) [inline makefile:35]: Why has this been added?

**Pull request closed without merging**